cliquet/statsd.py

```diff
@@ -0,0 +1,32 @@
+from functools import wraps
+import statsd
+
+
+def dummy_decorator(f):
+    """Just call the given function."""
+    @wraps(f)
+    def wrapper(*args, **kwargs):
+        return f(*args, **kwargs)
+    return wrapper
+
+
+class StatsdClient(object):
```

Comment:
I don't understand why this is a class since it only has class methods. A function would seem more appropriate.